### PR TITLE
import: add page

### PR DESCRIPTION
**Pull request by jeeftor**
ImageMagick utility Import -used to do screen capture for the command line.  Exists for both OSX and for Linux.  Where I work it used heavily.

### CODE REVIEW

File: pages/common/import.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# import
+
+> Capture some or all of an X server screen and save the image to a file. Part of ImageMagick library
```
Comment (agnivade) [inline pages/common/import.md]: Please make the second sentence as a new line starting with >

File: pages/common/import.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+- Capture the entire X server screen in the Postscript image format:
+
+`import -window root {{filename.postscript}}`
```
Comment (agnivade) [inline pages/common/import.md]: For png files, you have specified `output.png`, but here you mention `filename.postscript`. I would suggest to keep everything uniform, and mention `output` for all cases.

File: pages/common/import.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+- Capture contents of remote x server screen in the png image format:
+
+`import -window root -display {{remote.host}}:0.0 {{output.png}}`
```
Comment (agnivade) [inline pages/common/import.md]: Why is the remote host in the format of `remote.host` ? I think its better to mention it as `remote_host` ?

Comment (agnivade) [inline pages/common/import.md]: Also, if the output format is determined by the extension in the command, please put the `.png` part outside the tokens like this - {{output}}.png

File: pages/common/import.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+- Capture specific window with ID as displayed by `xwininfo`:
```
Comment (agnivade) [inline pages/common/import.md]: Please mention that the output will be in jpg format.

File: pages/common/import.md (hunk 1 above)
Comment (jeeftor) [inline pages/common/import.md]: Will do

**Comment by agnivade** [comment]
The changes look good. Please fix the Travis errors and we should be good to go.

**Comment by jeeftor** [comment]
Any update on when this will be merged in?

**Comment by agnivade** [comment]
Hey @jeeftor - sorry for the delay. It usually takes approval of 2 core members to get a PR merged. @waldyrious seems to be busy for some time. 

@ostera - can you merge this ? There are also some other pending PRs. Would be great if you have some time to take a look.

**Comment by agnivade** [comment]
Since @waldyrious seems to be inactive for some time and no other maintainers have any strong objections, I will go ahead and merge this. Anyways, we can make changes later if we decide. Better to keep the repo active rather than PRs lagging behind.

**Pull request merged**